src/core_plugins/kibana/public/home/components/recently_accessed.test.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { RecentlyAccessed, NUM_LONG_LINKS } from './recently_accessed';
+import { findTestSubject } from '@elastic/eui/lib/test';
+
+const createRecentlyAccessed = (length) => {
+  const recentlyAccessed = [];
+  let i = 0;
+  while(recentlyAccessed.length < length) {
+    recentlyAccessed.push({
+      label: `label${recentlyAccessed.length}`,
+      link: `link${recentlyAccessed.length}`,
+      id: `${i++}`
+    });
+  }
+  return recentlyAccessed;
+};
+
+test('render', () => {
+  const component = shallow(<RecentlyAccessed
+    recentlyAccessed={createRecentlyAccessed(2)}
+  />);
+  expect(component).toMatchSnapshot(); // eslint-disable-line
+});
+
+describe('more popover', () => {
+  test('should not be rendered when recently accessed list size is less than NUM_LONG_LINKS', () => {
+    const component = mount(<RecentlyAccessed
+      recentlyAccessed={createRecentlyAccessed(NUM_LONG_LINKS - 1)}
+    />);
+
+    const moreRecentlyAccessed = findTestSubject(component, 'openMoreRecentlyAccessedPopover');
+    expect(0).toBe(moreRecentlyAccessed.length);
```

Comment:
I think these assertions are reversed. I'd expect us to assert that the length of `moreRecentlyAccessed` would be 0 rather than the other way around.